ibm_catalog.json

```diff
@@ -182,6 +222,36 @@
                   }
                 }
               },
+              {
+                "key": "cspm_enabled",
+                "options": [
+                  {
+                    "displayname": "Enabled",
+                    "value": true
+                  },
+                  {
+                    "displayname": "Disabled",
+                    "value": false
+                  }
+                ]
+              },
+              {
+                "key": "app_config_crn"
+              },
+              {
+                "key": "config_aggregator_enterprise_id",
+                "virtual": true
+              },
```

Comment:
since this is a virtual inputs, it needs more info like the type, the description and default value. I also think you can remove the `config_aggregator_` prefix for the top level input:

```
{
    "key": "enterprise_id",
    "type": "string",
    "default_value": "__NULL__",
    "description": "If the account is an enterprise account, this value should be set to the enterprise ID (NOTE: This is different to the account ID).",
    "required": false,
    "virtual": true
},
```